source/dub/dub.d

```diff
@@ -1113,17 +1113,17 @@ class Dub {
 			recipe_callback = Optional callback that can be used to
 				customize the recipe before it gets written.
 	*/
-	void createEmptyPackage(NativePath path, string[] deps, string type,
+	void createEmptyPackage(NativePath path, string[] deps, string[string] depVers, string type, string typeVersion,
```

Comment:
I think it's a good idea to split type and typeVersion for the caller to explicitly put in a version, however this needs another overload only taking the type for backwards-compatibility purposes. (e.g. would pass `>=0.0.0` in that overload there)